packages/solidstart/src/server/utils.ts

```diff
@@ -31,3 +32,26 @@ export function isRedirect(error: unknown): boolean {
   const hasValidStatus = error.status >= 300 && error.status <= 308;
   return hasValidLocation && hasValidStatus;
 }
+
+/**
+ * Adds an event processor to filter out low quality transactions,
+ * e.g. to filter out transactions for build assets
+ */
+export function filterLowQualityTransactions(options: Options): void {
+  getGlobalScope().addEventProcessor(
+    Object.assign(
+      (event => {
+        if (event.type !== 'transaction') {
+          return event;
+        }
+        // Filter out transactions for build assets
+        if (event.transaction?.match(/^GET \/_build\//)) {
```

Comment:
Not for this PR but something to keep in mind in general: is it possible in solidstart to change the name of the build directory?

Reply:
It's generally possible to change, but vinxi has problems at build time when the build folder isn't named `_build`. I think for now this won't change but we can revisit this if it ever does.